build/build.proj

```diff
@@ -22,17 +22,22 @@
     <VsManProjectFile Include="$(RepositoryRootDirectory)src\VsixV3\ProjectSystemPackage\Microsoft.VisualStudio.ProjectSystem.Managed.vsmanproj" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <!-- NuGet will not automatically find the correct version of MSBuild. We need to specify the path. -->
+    <NugetArguments>-verbosity quiet -msbuildpath &quot;$(DevenvDir)\..\..\MSBuild\15.0\bin&quot;</NugetArguments>
```

Comment:
I'm not sure, @Pilchie, I just copied https://github.com/dotnet/roslyn/pull/14906, is that actually a bug?